utils/terrascript_client.py

```diff
@@ -1232,6 +1235,103 @@ def populate_tf_resource_s3_cloudfront(self, resource, namespace_info):
         for tf_resource in tf_resources:
             self.add_resource(account, tf_resource)
 
+    def populate_tf_resource_cloudwatch(self, resource, namespace_info):
+        account, identifier, common_values, \
+            output_prefix, output_resource_name = \
+            self.init_values(resource, namespace_info)
+
+        tf_resources = []
+        self.init_common_outputs(tf_resources, namespace_info,
+                                 output_prefix, output_resource_name)
+
+        # ecr repository
+        # Terraform resource reference:
+        # https://www.terraform.io/docs/providers/aws/r/
+        # cloudwatch_log_group.html
+        values = {
+            'name': identifier,
+            'tags': common_values['tags'],
+            'retention_in_days':
+                self._get_retention_in_days(
+                    common_values, account, identifier)
+        }
+
+        region = common_values['region'] or self.default_regions.get(account)
+        if self._multiregion_account_(account):
+            values['provider'] = 'aws.' + region
+        log_group_tf_resource = aws_cloudwatch_log_group(identifier, **values)
+        tf_resources.append(log_group_tf_resource)
+        output_name = output_prefix + '[log_group_name]'
+        output_value = '${' + log_group_tf_resource.fullname + '.name}'
+        tf_resources.append(output(output_name, value=output_value))
+        output_name = output_prefix + '[aws_region]'
+        tf_resources.append(output(output_name, value=region))
+
+        # iam resources
+        # Terraform resource reference:
+        # https://www.terraform.io/docs/providers/aws/r/iam_access_key.html
+
+        # iam user for log group
+        values = {
+            'name': identifier,
+            'tags': common_values['tags'],
+            'depends_on': [log_group_tf_resource]
+        }
+        user_tf_resource = aws_iam_user(identifier, **values)
+        tf_resources.append(user_tf_resource)
+
+        # iam access key for user
+        tf_resources.extend(
+            self.get_tf_iam_access_key(
+                user_tf_resource, identifier, output_prefix))
+
+        # iam user policy for log group
+        policy = {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Action": [
+                        "logs:CreateLogStream",
+                        "logs:DescribeLogStreams",
+                        "logs:PutLogEvents",
+                        "logs:GetLogEvents"
```

Comment:
The IAM user does not need `logs:GetLogEvents` permission.

Reply:
removed